### ISSUE TITLE
Screenreader does not announce the number of listed versions

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#953]
### Problem

When the version picker box is open, the screenreader is unable to determine the number of version list items. Thus a user cannot determine their progress through the list.
https://github.com/readthedocs/sphinx_rtd_theme/blob/5ba05cd1faa2ce22723b437d2b1d0f265dfcd225/sphinx_rtd_theme/versions.html

### Reproducible Project

https://docs.citusdata.com/en/stable/ 

#### Error Logs/Results

Here is a video demonstrating the problem: https://www.youtube.com/watch?v=-OvNZkY7-gA

#### Expected Results

The version picker should conform to [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) and help the screenreader announce the number of list items.

### Environment Info

- Python Version: 3.7.7
- Sphinx Version: 1.4.9
- RTD Theme Version: 0.2.4

**Comment by stsewd** [comment]
Not sure what would be the solution for this. They are inside a definition list, so I guess the solution is something else.
